src/plugins/intel_gpu/src/graph/fake_convert.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "fake_convert_inst.h"
+#include "fake_convert_shape_inference.hpp"
+
+#include "primitive_type_base.h"
+#include "intel_gpu/runtime/error_handler.hpp"
+#include "json_object.h"
+#include <string>
+
+namespace cldnn {
+GPU_DEFINE_PRIMITIVE_TYPE_ID(fake_convert)
+
+layout fake_convert_inst::calc_output_layout(fake_convert_node const& node, kernel_impl_params const& impl_param) {
+    return calc_output_layouts<ov::PartialShape>(node, impl_param)[0];
+}
+
+template<typename ShapeType>
+std::vector<layout> fake_convert_inst::calc_output_layouts(fake_convert_node const& node, kernel_impl_params const& impl_param) {
+    auto desc = impl_param.typed_desc<fake_convert>();
+    auto input_layout = impl_param.get_input_layout(0);
+    auto scale_layout = impl_param.get_input_layout(1);
```

Comment:
Please use const reference here to avoid data copying

Reply:
Fixed it.